cobalt/dom/performance_lifecycle_timing.idl

```diff
@@ -25,4 +25,6 @@ interface PerformanceLifecycleTiming : PerformanceEntry {
   readonly  attribute DOMHighResTimeStamp appDeeplink;
   readonly  attribute DOMString  currentState;
   readonly  attribute DOMString  lastState;
+
```

Comment:
Please add a comment here and on the `appStar` as well for those being non-spec. 

There should actually be a link to the DOM spec in the header comments here as well, as we do elsewhere in the IDL files.

Reply:
Yeah i realized that after i left the comment. We should probably at least leave a comment at the top that it's entirely non-spec. 

Also unsure but there's a related `VisibilityStateEntry` now https://html.spec.whatwg.org/multipage/interaction.html#the-visibilitystateentry-interface

I'm not sure if that existed when we did the original design.